src/components/Reusable/Button.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+type ButtonProps = {
+  title: string;
+  className?: string;
+  type?: "button" | "submit" | "reset" | undefined;
+  handleClick?: () => void;
+};
+
+const Button = ({ type, title, className, handleClick }: ButtonProps) => {
```

Comment:
Component Button is very generic, please rename it to PrimaryButton, SecondaryButton or TertiaryButton depending on how much this resuable component will be used in the project.